components/fxa-client/src/commands/send_tab.rs

```diff
@@ -77,19 +78,15 @@ pub struct TabHistoryEntry {
 
 #[derive(Clone, Serialize, Deserialize)]
 pub struct PrivateSendTabKeys {
-    public_key: Vec<u8>,
-    private_key: Vec<u8>,
+    ec_key: EcKeyComponents,
     auth_secret: Vec<u8>,
```

Comment:
Similar to what push does and what fxa-client does, should we put a version number in this struct somewhere so that we can evolve it in future in a backwards-compatile way?

Reply:
If you wanted to be a little more explicit you could call the field `p256key` or similar, for a bit of symmetry with how you use `p256dh` to name the public key when saving to the server.